### ISSUE TITLE
Not interacting

### ISSUE THREAD

**Issue by fahadammar** [seosh817/CircularSeekBar#1]
Following the guideline, I have implemented the circular progress bar. But as I touch and try to roll it, it's not being interactive. I have seen your example project, when I touch and roll forward, it progresses, but currently it's not being interactive as I have implemented. Need help.

**Comment by seosh817** [comment]
Hi @fahadammar ,

Thank you for reporting the issue. If you want to make the CircularSeekbar interactive, you have to change the interactive property. Could you try setting the value of the interactive property to true?

```
    <com.seosh817.circularseekbar.CircularSeekBar
        android:id="@+id/circular_seek_bar"
        ...
       app:circularSeekBar_interactive="true"
 />
```

**Comment by fahadammar** [comment]
Ok It worked now. I really need help.

I'm using the viewpager and on first fragment, the CircularSeekbar is added in the XML, when I switch to 2nd fragment and comes back again, the app crashes.

These are the crashes logs:
**Process: com.example.volumeboosterdummy, PID: 22354
    java.lang.IllegalStateException: The specified child already has a parent. You must call removeView() on the child's parent first.
        at android.view.ViewGroup.addViewInner(ViewGroup.java:5264)
        at android.view.ViewGroup.addView(ViewGroup.java:5093)
        at android.view.ViewGroup.addView(ViewGroup.java:5030)
        at android.view.ViewGroup.addView(ViewGroup.java:5002)
        at com.seosh817.circularseekbar.CircularSeekBar.onAttachedToWindow(CircularSeekBar.kt:366)**
        
        Need guidance how to solve this issue ???
        
        Looking forward to the reply thank you!

**Comment by seosh817** [comment]
Thanks for reporting this! I checked this problems and confirmed that the bug was caused by not calling the removeView method.
I will pick this up as soon as possible.

**Comment by seosh817** [comment]
Version 1.0.2 has been released.

**Comment by seosh817** [comment]
Hi @fahadammar,

Can you tell me if the problem was resolved in Version 1.0.2? 
thank you :)

**Comment by fahadammar** [comment]
Yes the issue/bug is fixed/resolved. Thank you so much for all this!!
